### ISSUE TITLE
I don't understand the slicing of the validation set here！

### ISSUE THREAD

**Issue by ddd433** [ChristopherSims/FMRI_ADHD_Classification#2]
Training_df = self.df.iloc[0:split,:]
        Validation_df = self.df.iloc[0:(len(self.df)-split),:]

**Comment by ChristopherSims** [comment]
This splits the data into train and val. You can modify to code to use a Keras loader with partitions, but it has the same effect.
That is an error in the code line, it should read:
**Validation_df = self.df.iloc[split:len(self.df),:]**
The performance of the model is the same (tested on my local dataset)

**Comment by ddd433** [comment]
Has the paper been published? Can you give me a link?Thank u!

**Comment by ChristopherSims** [comment]
on the mainpage: https://arxiv.org/abs/2205.11993

**Comment by ddd433** [comment]
Which sites from the ADHD-200 dataset did you use for your data, and how many subjects you use in your model?In multimodal training, is there any correlation between MRI data and fMRI data? I am looking forward your reply,Thank u!

**Comment by ChristopherSims** [comment]
datasets are noted in the paper. fMRI and MRI are separate, the paper talk about it
